components/tabs/tabs.js

```diff
@@ -169,11 +169,14 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 				transition: margin-top 200ms ease-out;
 			}
 
-			d2l-tab-internal {
+			d2l-tab-internal, ::slotted(d2l-tab) {
```

Comment:
I handled this in tabs rather than in `TabMixin` since it felt like the concerns of the overall `d2l-tabs` rather than the individual tab.

I'll have to update this selector so that it would target custom tabs too.